packages/hardhat/deploy/99_generateTsAbis.ts

```diff
@@ -24,7 +27,22 @@ function getContractNames(path: string) {
     .map(dirent => dirent.name.split(".")[0]);
 }
 
-const DEPLOYMENTS_DIR = "./deployments";
+function getInheritedFunctions(sources: { [key: string]: any }, contractName: string) {
```

Comment:
`sources : Record<string, any>` just to stay consistent 🙌

Reply:
fixed here: https://github.com/FilipHarald/scaffold-eth-2-sandbox/commit/4d98920f33bdb0cb67cf325f398e9580cf041d3d